connector/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/LiteralValueProtoConverter.scala

```diff
@@ -99,20 +96,6 @@ object LiteralValueProtoConverter {
       case proto.Expression.Literal.LiteralTypeCase.DAY_TIME_INTERVAL =>
         expressions.Literal(lit.getDayTimeInterval, DayTimeIntervalType())
 
-      case proto.Expression.Literal.LiteralTypeCase.ARRAY =>
-        val literals = lit.getArray.getValuesList.asScala.toArray.map(toCatalystExpression)
-        CreateArray(literals)
-
-      case proto.Expression.Literal.LiteralTypeCase.STRUCT =>
```

Comment:
Where do these go?

Reply:
This was removed per https://github.com/apache/spark/pull/39047#discussion_r1046801784. See also https://github.com/apache/spark/pull/39047#discussion_r1047897612